swagger.yaml

```diff
@@ -41,4 +181,50 @@ components:
           type: integer
           format: int32
         message:
-          type: string
\ No newline at end of file
+          type: string
+
+    Project:
+      type: object
+      description: Project object
+      required:
+        - name
+      properties:
+        id:
+          type: string
+          format: uuid
+        name:
+          type: string
+        tocomanId:
+          type: integer
+        metadata:
+          $ref: '#/components/schemas/Metadata'
+
+    Task:
+      type: object
+      description: Task object
+      properties:
+        tocomanId:
+          type: integer
+        name:
+          type: string
```

Comment:
This is not used anywhere, should it be in this PR?

Reply:
Tocoman id? Hmm probably not, when making it i assumed it's going to be useful later, but probably not

Reply:
removed